### ISSUE TITLE
[Feature Request] - Hermaphrodite Self-Fertilization + Stat

### ISSUE THREAD

**Issue by guardiaforest** [Kitteh6660/Corruption-of-Champions-Mod#261]
There's a self-fuck scene with Shouldra on the ruins for hermaphrodite characters, but it doesn't lead to pregnancy.

My idea is for a hermaphrodite character with a long penis and high lust to have a self-fuck option besides the masturbation/meditation option. This scene wouldn't require Shouldra and this scene could lead to pregnancy through self-ferilization, giving the player an achievment.

The number of self-fertilizations would be tracked through a secondary stat.

The baby/child could be male, female or hermaphrodite, with a chance of being pure human or taking some morph features depending on the character classification. Every time that happens the player would be visited by an adoption agency that would convince the player to hand in their baby and maybe would leave a reward for the player.

**Comment by JustGregory** [comment]
On the latter part ... really? REALLY? Send away your own CHILD???
...
*sigh* -- "It's only a game," I keep telling myself. But I'm concerned about the subliminal message being elicited in that last sentence. "Pop out a child, give it up for adopt, get a fast reward."
...
~ In all seriousness, though; I have thoughts on other aspects of this. Personally, I'd like the option to go the other way, to *count* offspring and incidentals in camp population, even if they should not be active participants. Heck, why not the option to have some of Tamani's kids running around the camp? If I can't kill Tamani (no matter how many times I encounter the little skank) maybe at some the children might show up, at least in flavor texts. (There's also "silly mode" potential, in that as many female monstergirls as the player could vaginally penetrate, at some point one or more of these monster girls should show up to "demand you do the right thing" and take care of her during a pregnancy term.... They don't have to be followers, they don't have to be lovers, but -- some allowance for incidental NPCs might be permissible.)
I also have the idea that hermaphrodites should not be fertile, at least not *self-fertilizable*, so that they cannot have any pregnancy. I am by no means expert at anything genetics-related, but I do think it should be highly improbable. At least consider such self-cest a disable-able option overall, and asked of the player who chooses the hermaphrodite character route, at new-game.

**Comment by guardiaforest** [comment]
I thought only about sending them away because of it would be much easier to program something like that, (also, a lot of your children are sent away depending on which NPC has impregnated you or has been impregnated by you), but if possible, it would be interesting to have them at camp or somewhere you could visit them and maybe even interact with them, but I think it would be better to take one step at a time

About fertility, on real life it is improbable that an intersex person (hermaphrodite) would be fertile for a single sex alone, there has been very few cases of supposed hermaphrodites that were fertile both ways; It would take a human chimera (person with two or more sets of chromossomes resulting from a fusion of sibling zygotes), with a XX on one set and XY on the other, them that person would need to develop both male and female genitalia and have regular hormone levels (very unlikely on that situation) to bear a child. In the case that we get a XX-XY-chimera intersex person with both set of fully functional genitals and that person is able to self-fertilize, the child would get half of each set of chromossomes. In the game though, most hermaphrodites are fertile both ways independently of background, so self-fertilization should not be a problem

I agree that this should be a disable-able option though, and the ghostly aided self-cest with Shouldra should be under that option as well

**Comment by JustGregory** [comment]
It's understandable that it would be easier to program; but not outside the realm of possibility. I agree with having children in camp; the general idea I've had all along is, the player is basically **building a small village** with the various improvements to the camp area; why not have a wing of the cabin/house (or a separate building) be a nursery?

But, back to the OP point. So then, if I understand correctly, you would be wanting herm-pregnancy allowable, primarily for the novelty factor, due to the rarity in real life? regardless of whether the child is kept or not?

**Comment by guardiaforest** [comment]
Indeed, it's primarily for novelty reasons, though having it affect the game beyond that would surely be a plus.

Side note: congratulations on eliminating Tamani :)

**Comment by Stadler76** [comment]
Uhm, wouldn't that be incest? Just asking

**Comment by JustGregory** [comment]
AFAIK, incest is _with another individual_ -- what is discussed would be termed "self-cest" ... someone correct this if I'm wrong.

**Comment by guardiaforest** [comment]
Indeed, it would be selfcest (the resulting children would look like one resulting from either incest, twincest or clonecest though)
